uui-editor/src/plate/plugins/attachmentPlugin/AttachmentBlock.tsx

```diff
@@ -37,14 +37,14 @@ export function AttachmentBlock(props: any) {
     const isReadonly = useReadOnly();
 
     const { element, editor, children } = props;
-    const [fileName, setFileName] = useState(element.data.fileName || element.fileName);
+    const [name, setName] = useState(element.data.name || element.name);
 
     const changeName = (name: string) => {
         setElements(editor, {
             ...element,
             data: {
                 ...element.data,
-                fileName: name,
+                name: name,
```

Comment:
it's not safe to change the slate state object fields, because it will cause integration issues with already created states from old version. Let's leave here `fileName` field as earlier.

Reply:
Fixed